frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua

```diff
@@ -246,15 +325,25 @@ return {
     _Y_el = {
         "Υ",
         northeast = "Ύ",
-        northwest = "Y",
-        north = "Ϋ",
+        north = "Y",
+        west = "Ὑ",
+        south = "Ϋ",
+        southeast = "Ὕ",
+        southwest = "Ὗ",
     },
     _y_el = {
         "υ",
         northeast = "ύ",
-        northwest = "Y",
-        north = "ΰ",
-        west = "ϋ",
+        north = "y",
+        northwest = "ὺ",
+        east = "ὐ",
+        west = "ὑ",
+        south = "ΰ",
```

Comment:
I'm sorry, but I've noticed some lines that need to be changed.

The combination "upsilon with acute and diaeresis" (ΰ) there is twice. So line 341 should be deleted
`south = "ΰ",`